bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -1836,49 +1946,25 @@ public void setPositionTrack(String tracknr) {
 
     protected void seek(String unit, @Nullable String target) {
         if (target != null) {
-            Map<String, String> inputs = new HashMap<>();
-
             try {
-                inputs.put("InstanceID", "0");
-                inputs.put("Unit", unit);
-                inputs.put("Target", target);
+                executeAction(SERVICE_AV_TRANSPORT, ACTION_SEEK,
+                        Map.of("InstanceID", "0", "Unit", unit, "Target", target));
             } catch (NumberFormatException ex) {
```

Comment:
same question

Reply:
I agree, I don't know why this try/catch was there. I can suppress it.

Reply:
Suppressed